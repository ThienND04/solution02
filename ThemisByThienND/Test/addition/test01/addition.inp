10
44134 57792 5662 999953507
42592 57239 6379 999965456
44375 57372 6515 999949319
43419 55729 6506 999933878
43399 56288 6881 999944047
43140 54752 7192 999917138
43732 56516 7368 999994320
42413 56790 6418 999924194
44234 54530 6493 999984681
42526 57308 7110 999966666
