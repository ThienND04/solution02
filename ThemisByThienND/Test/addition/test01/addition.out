0
32151
30175
31761
29744
0
109838
32336
31930
57711
